src/main/java/salesforce/utils/SalesforceUrl.java

```diff
@@ -17,8 +17,12 @@
  */
 public class SalesforceUrl {
     private String loginPage;
-    private String salesPage;
-    private String opportunitiesPage;
+    private String salesLightningPage;
+    private String opportunitiesLightningPage;
+
+    private String salesClassicPage;
+    private String opportunitiesClassicPage;
```

Comment:
They would be e.g. "loginUrlPage", "salesLightningUrlPage;" it would be for not generate a confusion with pages

Reply:
but it, I manage through the map of transporter